antenna/ext/gcs/crashstorage.py

```diff
@@ -170,3 +185,33 @@ def save_crash(self, crash_report):
 
         self.save_dumps(crash_id, dumps)
         self.save_raw_crash(crash_id, raw_crash)
+
+    def load_crash(self, crash_id):
+        """Load crash data."""
+        raw_crash = {}
+        dumps = {}
+
+        raw_crash_key = self._get_raw_crash_path(crash_id)
+        try:
+            raw_crash = json.loads(self._load_file(raw_crash_key))
+        except NotFound as exc:
+            raise CrashIDNotFound(f"{crash_id} not found") from exc
+
+        dump_names_path = self._get_dump_names_path(crash_id)
+        try:
+            dump_names = json.loads(self._load_file(dump_names_path))
+        except NotFound:
+            logger.error("raw_crash exists, but dump_names doesn't exist")
```

Comment:
This is in the `load_crash` code and this situation would be super weird--this should never happen. I put this here because the linter was annoyed with a `pass` (which is valid). I can change it to raise an error.